src/s3-resource-selector/s3-modal/index.tsx

```diff
@@ -1,7 +1,7 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 import React, { useReducer, useRef } from 'react';
-import InternalBreadcrumbGroup from '../../breadcrumb-group/internal';
+import { BreadcrumbGroupImplementation } from '../../breadcrumb-group/implementation';
```

Comment:
Changed to`/implementation` import to make this component use non-widgetized version